lookout/style/format/features.py

```diff
@@ -414,24 +432,26 @@ def _add_noops(vnodes: Sequence[VirtualNode], path: str) -> List[VirtualNode]:
                                       y=CLASS_INDEX[CLS_NOOP], path=path))
         return result
 
-    def _get_internal_type_index(self, vnode: VirtualNode) -> int:
-        role_index = -1
-        if vnode.node:
-            role = vnode.node.internal_type
-            if role in self.roles.INTERNAL_TYPES_INDEX:
-                role_index = self.roles.INTERNAL_TYPES_INDEX[role]
-        return role_index
+    def _get_internal_type_index(self, node: Union[bblfsh.Node, VirtualNode]) -> int:
+        internal_type_index = -1
+        bblfsh_node = node if isinstance(node, bblfsh.Node) else node.node
```

Comment:
I met an `AttributeError` here because in some cases node is `gopkg.in.bblfsh.sdk.v1.uast.generated_pb2.Node` class so `isinstance(node, bblfsh.Node)` retruns `False`

Reply:
@vmarkovtsev wasn't this fixed in https://github.com/bblfsh/client-python/pull/124?
@waren I'll change the test (use virtual node to test and default to bblfsh